LICENSE.unlicense

```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```

Comment:
There's no need to include the source of this license here

Reply:
done